test/integration/app2/inspec/app2_spec.rb

```diff
@@ -65,17 +60,16 @@
   end
 end
 
-describe file('/var/lib/formsender/Dockerfile') do
-  it { should exist }
-end
-
-describe docker.images.where { tag == 'latest' } do
-  it { should exist }
-end
-
 describe docker_container('support.osuosl.org') do
   it { should exist }
   it { should be_running }
   its('image') { should eq 'formsender:latest' }
   its('ports') { should eq '0.0.0.0:8086->5000/tcp' }
 end
+
+describe http(
+  'http://127.0.0.1:8086/server-status',
+  headers: { 'Host' => 'support.osuosl.org' }
+) do
```

Comment:
You shouldn't need to set the vhost, just do this instead please:
```suggestion
describe http 'http://127.0.0.1:8086/server-status' do
```